web/angular-wallet/src/locales/en.json

```diff
@@ -609,6 +609,7 @@
   "next": "Next",
   "next_block": "Next Block",
   "next_page": "Next Page",
+  "network": "Network",
```

Comment:
@jjos2372 is this a new string? If so, do we need to add it to all locales?

Reply:
yeah, it is new and I think I added some more. It is probably a good idea to add on all locales, maybe @ohager knows how to automate this.

Anyway it just defaults to English if not found when I tested here.

Reply:
I was wrong, new strings do not fall back for English, we need to add them for all languages.

Reply:
My solution to simplify the translation is the i18n Manager App. That way I can add new items in seconds and even let them translate using Google API (without any additional costs)